modules/ingestor/src/endpoints/advisory.rs

```diff
@@ -13,85 +19,104 @@ use trustify_module_graph::graph::Graph;
         ("advisory_format" = String, Path, description = "Format of the submitted advisory document (`casf`, `osv`, ...)"),
     ),
     responses(
-        (status = 200, description = "Upload a file"),
-        (status = 400, description = "The file could not be parsed as an advisory document"),
+        (status = 201, description = "Upload a file"),
+        (status = 400, description = "The file could not be parsed as an advisory"),
     )
 )]
-#[post("/advisories/{advisory_format}")]
+#[post("/advisories")]
 /// Upload a new advisory
 pub async fn upload_advisory(
-    graph: web::Data<Graph>,
-    req: HttpRequest,
-    path: web::Path<String>,
+    service: web::Data<IngestorService>,
     payload: web::Payload,
+    web::Query(UploadAdvisoryQuery { location, format }): web::Query<UploadAdvisoryQuery>,
```

Comment:
Moving to query params instead of path param?  I'm fine with that. We should probably start a `Tenets` document or discussion to ensure we keep consistent and how we decide which option to choose.

Reply:
https://github.com/trustification/trustify/discussions/122

Reply:
Cool. I don't have a strong opinion either way, I was just reusing the Query struct.